api/ChattO/API/Controllers/UserController.cs

```diff
@@ -37,60 +38,37 @@ public async Task<IActionResult> Create([FromBody] CreateAccountsRequest request
     [ProducesResponseType(StatusCodes.Status404NotFound)]
     public async Task<IActionResult> GetById([FromRoute] Guid id)
     {
-        return Ok();
+        var userResult = await Mediator.Send(new GetDetailsAppUser.Query { Id = id });
+        if (!userResult.IsSuccessful)
+            return HandleResult(userResult);
+
+        var response = Mapper.Map<UserDetailsResponse>(userResult.Data);
+        response.Organization = Mapper.Map<GetDetailsOrganizationResponse>(userResult.Data.Organization);
+        response.ProfileImage = Mapper.Map<FileResponse>(userResult.Data.ProfileImage);
```

Comment:
I believe it will work in single map operation "Mapper.Map<UserDetailsResponse>(userResult.Data)"

Reply:
Fixed